mobile/library/cc/engine_builder.h

```diff
@@ -77,20 +76,17 @@ class EngineBuilder {
   EngineBuilder& setNodeLocality(const NodeLocality& node_locality);
   // Adds an ADS layer. Note that only the state-of-the-world gRPC protocol is supported, not Delta
   // gRPC.
-  EngineBuilder&
-  setAggregatedDiscoveryService(const std::string& address, const int port,
-                                std::string jwt_token = "",
-                                int jwt_token_lifetime_seconds = DefaultJwtTokenLifetimeSeconds,
-                                std::string ssl_root_certs = "");
+  EngineBuilder& setAggregatedDiscoveryService(std::string address, const int port,
+                                               std::string jwt_token = "",
+                                               int jwt_token_lifetime_seconds = 0,
```

Comment:
is the reason that this (and the timeouts below) were changed from Default to 0 so that the other languages can just pass in 0 instead of having to know the defaults?